custom_components/mammotion/lawn_mower.py

```diff
@@ -151,6 +155,7 @@ def activity(self) -> LawnMowerActivity | None:
     async def async_start_mowing(self, **kwargs: Any) -> None:
         """Start mowing."""
         if kwargs:
+            await self.async_cancel()
```

Comment:
What's going to happen if it doesn't need to cancel? Nothing?

Reply:
yes, cancel function is limited to running the "cancel_job" command when Luba is stopped in MODE_PAUSE. 

There was still the question of if the command should run for MODE_READY but on LUBA 2 at least I still only see 
MODE_PAUSE => active_task
MODE_READY => no active task
(MODE_RETURN need to cancel recharge and check resulting status)

Reply:
Yeah I don't think it needs to be called on MODE_READY

Trying to remember what my luba 1 does.